scipy/signal/signaltools.py

```diff
@@ -1439,10 +1439,14 @@ def wiener(im, mysize=None, noise=None):
     >>> import numpy as np
     >>> import matplotlib.pyplot as plt
     >>> from pylab import *
-    >>> x = np.random.random((40,40))         #Create a random image
-    >>> plot1 = plt.imshow(x)                 #Plot the image
-    >>> #Show image filtered with 5px by 5px window
-    >>> plot2 = plt.imshow(wiener(x, (5, 5)))
+    >>> img = np.random.random((40,40))    #Create a random image
+    >>> #Filter image with 5px by 5px window
+    >>> filtered_img = wiener(img, (5, 5))
+    >>> plt.figure()
+    >>> f, arr = plt.subplots(1, 2)
```

Comment:
You can also add figure size to control the whitespace if the default is too big. See the example here https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.fourier_gaussian.html